.github/pull_request_template.md

```diff
@@ -0,0 +1,8 @@
+
+
+----
+
+By submitting this pull request, I promise that:
+
+- [ ] I have read the [Pull Request + Code Style Guidelines](https://github.com/webcompat/webcompat.com/blob/master/docs/pr-coding-guidelines.md) thoroughly.
+- [ ] I ensure my pull request follows each and every point.
```

Comment:
I'm not sure about this second line, is this a common pattern other projects follow?

Reply:
Drive-by comment:
The template if I understood will be displayed each time someone create a pull request. It will become monotonous for people contributing often to have to this message.

On the other hand it could be useful to have in the template something guiding the person to improve the pull request.

```md
This PR fixes issue `#nnnn`

## Proposed PR background
(put here comments helping us to understand the PR.)

PS: Make sure your PR is following [Pull Request + Code Style Guidelines](https://github.com/webcompat/webcompat.com/blob/master/docs/pr-coding-guidelines.md)
```

Reply:
> I'm not sure about this second line, is this a common pattern other projects follow? 

I just it in a Repo where people continuously don’t stick to the rules. Removed it with the latest commit.